pom.xml

```diff
@@ -335,7 +560,7 @@
                         <sourceDirectory>src/main/java</sourceDirectory>
                     </sourceDirectories>
                     <configLocation>${maven.multiModuleProjectDirectory}/config/checkstyle/checkstyle.xml</configLocation>
-                    <propertyExpansion>config_loc=${maven.multiModuleProjectDirectory}/config/checkstyle</propertyExpansion>
+                    <propertyExpansion>config_loc=${maven.multiModuleProjectDirectory}/config/checkstyle</propertyExpansion> <!-- Needs to be synchronized with the caching config for checkstyle above -->
```

Comment:
Good to add this note. Can we say "in the Develocity Caching Configuration" otherwise I think "above" is too broad.